src/main/java/teammates/common/datatransfer/questions/FeedbackRankRecipientsQuestionDetails.java

```diff
@@ -215,6 +229,11 @@ public String getQuestionResultStatisticsCsv(
         StringBuilder fragments = new StringBuilder();
         Map<String, List<Integer>> recipientRanks = generateOptionRanksMapping(responses);
 
+        boolean isAvgExclSelfShown = shouldSelfBeExcludedFromRankings(question);
+        Map<String, List<Integer>> recipientRanksExcludingSelf = new HashMap<>();
+        if (isAvgExclSelfShown) {
+            recipientRanksExcludingSelf = getRecipientRanksExcludingSelf(responses);
+        }
```

Comment:
This code is being duplicated from `getQuestionResultStatisticsHtml`. Can this be extracted into a method?